ImageNet/inference/DidiCloud_resnet50_1p4_ifx.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "v1.0",
+    "author": "InferenceX Team of Didi Cloud",
+    "authorEmail": "caijinping@didiglobal.com, yangjintao@didiglobal.com",
+    "framework": "ifx",
+    "codeURL": "https://github.com/didiyun/dawnbench-ifx",
+    "commitHash": "4bb843cc4d195e47fd722dc7f597020154f165fd",
+    "model": "ResNet50",
+    "hardware": "Didi Cloud [1 P4 / 16 GB / 8 vCPU]",
+    "latency": 1.5439,
+    "cost": 3.21e-7,
```

Comment:
I am not familiar with Didi cloud. Please give me a link to pricing information and the instance type so I can verify the cost.

Reply:
> I am not familiar with Didi cloud. Please give me a link to pricing information and the instance type so I can verify the cost.

Thank you for your reply. 
You can get the price information by visiting [Didi cloud gpu details](https://www.didiyun.com/production/gpu.html#charge)，Also you can create an instance by visiting [Didi cloud create an innstance](https://app.didiyun.com/new/#/dc2/add?cpuNum=8&memorySize=17179869184&bandwidth=1024&srvType=ebs.gpu.p4&zoneId=&regionId=).

![image](https://user-images.githubusercontent.com/8033689/59741115-9c849980-929c-11e9-9376-6cd82bb64548.png)

Reply:
Thanks looks good to me!